policy-controller/core/src/http_route.rs

```diff
@@ -33,6 +44,32 @@ pub struct InboundHttpRouteRule {
     pub filters: Vec<InboundFilter>,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct OutboundHttpRouteRule {
+    pub matches: Vec<HttpRouteMatch>,
+    pub backends: Vec<Backend>,
+}
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum Backend {
+    Addr(WeightedAddr),
+    Dst(WeightedDst),
+    InvalidDst(WeightedDst),
+}
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct WeightedAddr {
+    pub weight: u32,
+    pub addr: IpAddr,
+    pub port: NonZeroU16,
+}
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct WeightedDst {
```

Comment:
What is a Dst? Is it a Service? Or what?

We're going to need structured backend references in addition to just authorities.